### PR TITLE
Make project compatible with msys2

### PR DESCRIPTION
**Pull request by masci**
Make it easy to build the project for those not familiar with Windows

### CODE REVIEW

**Comment by masci** [comment]
Indeed that was my bad, Msys2 correctly defines `_WIN32` already

File: three/three.def [hunk 1]
```diff
@@ -1,6 +1,6 @@
-LIBRARY datadog-agent-two
+LIBRARY datadog-agent-three
```
Comment (derekwbrown) [inline three/three.def]: I think we can get rid of the `def` files altogether; I don't think we're using them any more. But that can be a separate PR; it's not restricted to this change

Comment (masci) [reply three/three.def]: Will do, thanks for confirming I suspected something like this since the typo seemed too big to get unnoticed
